client/components/PubHistoryViewer/PubHistoryViewer.stories.tsx

```diff
@@ -311,6 +311,11 @@ export const initialHistoryData = {
 	timestamps: editTimestamps,
 	currentKey: 14678,
 	latestKey: 14678,
+	isViewingHistory: false,
```

Comment:
This is nice to have but curious why you made this change — some new TS error?

Reply:
pretty much, when trying to render something in storybook it was erroring out without those types being present